### PR TITLE
Custom auto-correlations

### PR DESCRIPTION
**Pull request by damonge**
Makes use of the new `custom_auto` flag at the C_ell level

### CODE REVIEW

File: xcell/mappers/mapper_dummy.py [hunk 1]
```diff
@@ -168,6 +171,13 @@ def get_nl_coupled(self):
                 self.nl_coupled[-1] += self.noise_level
         return self.nl_coupled
 
+    def get_cl_coupled(self):
```
Comment (carlosggarcia) [inline xcell/mappers/mapper_dummy.py:174]: Is it possible to add a test for this method alone?

**Review by carlosggarcia — CHANGES_REQUESTED** [review]
Minor comments. Maybe add a test for `get_cl_coupled` in `MapperDummy`

File: xcell/cls/cl.py [hunk 2]
```diff
@@ -163,15 +163,33 @@ def get_cl_file(self):
...
             wins = w.get_bandpower_windows()
-            cl_cp = nmt.compute_coupled_cell(f1, f2)
-            cl = w.decouple_cell(cl_cp)
+
+            # Compute power spectrum
+            # If auto-correlation, compute noise and,
+            # if needed, the custom signal power spectrum.
+            auto = self.tr1 == self.tr2
+            # Noise
             if self.tr1 == self.tr2:
```
[the remark this answers is not in the document]
Comment (damonge) [reply xcell/cls/cl.py]: duh... thanks, done

File: xcell/mappers/mapper_dummy.py (hunk 1 above)
Comment (damonge) [reply xcell/mappers/mapper_dummy.py:174]: yep, done

**Comment by damonge** [comment]
ok, back to you @carlosggarcia !

**Review by carlosggarcia — APPROVED** [review]
LGTM

**Pull request merged**